packages/db/src/test-utils.ts

```diff
@@ -32,6 +32,7 @@ export const generateLocalTestDb = async (
       ...localStackConnectionEnv,
       PG_DATABASE: testDbName,
       migrationDir,
+      // KNEX_DEBUG: 'true',
```

Comment:
Is this unintentional?

Reply:
Yeah, just leftover from testing. Though there wasn't an easy way to set this when using `generateLocalTestDb` for test setup.

Reply:
https://github.com/nasa/cumulus/pull/2091/commits/730f395675f7d834da425e71cb2a85637bcfe6b6